client/components/InputForm.jsx

```diff
@@ -24,11 +24,10 @@ export default class InputForm extends React.Component {
           type="text"
           rows="30"
           cols="150"
-          onChange={this.onChange}
-          value={text}>
+          onChange={this.props.onChange}
+          value={this.props.inputText}>
```

Comment:
this.props.inputText is never created/passed down from <TextView />. Did you mean to rename 'text' (in TextView.jsx line 33) to 'inputText'?